UploadDaemon_Test/SymbolAnalysis/SymbolCollectionResolverTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UploadDaemon.Configuration;
+
+namespace UploadDaemon.SymbolAnalysis
+{
+    [TestFixture]
+    class SymbolCollectionResolverTest
+    {
+        private readonly GlobPatternList someAssemblyPatterns = new GlobPatternList(new List<string> { "*" }, new List<string> { });
```

Comment:
the name is confusing for me. How about `includeAllAssembliesPattern`? Reflects better what it does and makes the tests easier to read